### PR TITLE
Detect more bots in `dim-bots`

### LINKED ISSUE #4952: Dim more bots
### Description

Extend `dim-bots` to include more bots:

- @actions-user: belongs to GitHub. it seems to be used by GitHub's example initially, and now a [widely used](https://github.com/search?o=desc&q=author%3Aactions-user&s=committer-date&type=Commits) placeholder besides `github-actions[bot]`.
- @web-flow: belongs to GitHub. See its bio
- @ImgBotApp: [usage](https://github.com/search?o=desc&q=author%3AImgBotApp&s=committer-date&type=Commits)
- @bors (@rust-lang bot user) and [bors](https://github.com/apps/bors) (GitHub app). Note these are "merge bots" in the sense that the duty of them is to merge commits, so they are already dimmed out by `mark-merge-commits-in-list`. But I think it makes sense enough to fold them via `dim-bots`.
- @Octomerger: as can be seen in https://github.com/github/docs

Adding more popular bots from https://github.com/marketplace is welcome.

### Screenshot

_No response_

### Example URL

N/A

### PR DESCRIPTION
**Pull request by cheap-glitch**
Resolves #4952 
Follow-up of #5139 

## Test URLs

 * https://togithub.com/rust-lang/rust/commits/master
 * https://togithub.com/OpenLightingProject/open-fixture-library/commits/c18c0b9caf5127637c56ff518623a0fd3effb738/.github/workflows/test.yaml

## Screenshots

image

image

### CODE REVIEW

File: source/features/dim-bots.tsx [hunk 1]
```diff
@@ -4,15 +4,24 @@ import * as pageDetect from 'github-url-detection';
...
+	'bors',
+	'imgbot',
+	'octomerger',
+	'renovate-bot',
+	'rust-highfive',
+	'rust-lang',
+	'scala-steward',
+	'snyk-bot',
+	'web-flow',
+].map(bot => `.commit-author[href$="?author=${bot}"]`);
```
Comment (cheap-glitch) [inline source/features/dim-bots.tsx:18]: Including the `?author=` part ensures we match the exact bot name (prefixing with a slash doesn't work in this case).

File: source/features/dim-bots.tsx [hunk 2]
```diff
@@ -4,15 +4,24 @@ import * as pageDetect from 'github-url-detection';
...
+
+commitSelectors.push('.commit-author[href$="%5Bbot%5D"]'); // Generic [bot] label
+
+const commitSelector = commitSelectors.join(',');
 
 const prSelector = [
-	/* Issues/PRs */
 	'.opened-by [href*="author%3Aapp%2F"]',
 	'.labels [href$="label%3Abot"]',
 ];
```
Comment (cheap-glitch) [inline source/features/dim-bots.tsx:27]: Should we also expand the list of PR selectors? Or is this impossible due to DOM limitations?

Comment (fregante) [reply source/features/dim-bots.tsx:27]: Up to @kidonng, who opened the original request

**Review by fregante — CHANGES_REQUESTED** [review]
Let's see if the other PR is reopened, then you can make further changes on top of that

https://github.com/refined-github/refined-github/pull/5139#issuecomment-991724829

File: source/features/dim-bots.tsx (hunk 2 above)
Comment (kidonng) [reply source/features/dim-bots.tsx:27]: I'm not sure what that means? It's looks fine as-is.

**Review by fregante — APPROVED** [review]

File: source/features/dim-bots.tsx [hunk 3]
```diff
@@ -4,15 +4,24 @@ import * as pageDetect from 'github-url-detection';
...
-const commitSelector = [
-	/* Commits */
-	'.commit-author[href$="%5Bbot%5D"]',
-	'.commit-author[href$="renovate-bot"]',
-	'.commit-author[href$="scala-steward"]',
-].join(',');
+const commitSelectors = [
+	'actions-user',
+	'bors',
+	'imgbot',
```
Comment (kidonng) [inline source/features/dim-bots.tsx:10]: 😅 Oops, I never read the actual list here and there are some error here, for example @imgbot should be @imgbotapp

Comment (cheap-glitch) [reply source/features/dim-bots.tsx:10]: Fixed in https://github.com/refined-github/refined-github/commit/79ef6d8f680ab39c17bb6ec4458b2228769e5fe8

File: source/features/dim-bots.tsx (hunk 2 above)
Comment (kidonng) [reply source/features/dim-bots.tsx:27]: @cheap-glitch perhaps you meant #5512 😄 sorry I'm dumb

File: source/features/dim-bots.tsx [hunk 4]
```diff
@@ -4,15 +4,24 @@ import * as pageDetect from 'github-url-detection';
...
+	'octomerger',
+	'renovate-bot',
+	'rust-highfive',
+	'rust-lang',
```
Comment (kidonng) [inline source/features/dim-bots.tsx:14]: And I never get to resolve this one on my to-do list (it's a mistake, @rust-lang is a organization and won't be a bot)

**Pull request merged**